Cargo.lock

```diff
@@ -3039,7 +3039,7 @@ dependencies = [
  "serde",
  "serde_json",
  "serial_test",
- "serial_test_derive",
+ "serial_test_derive 0.5.1",
```

Comment:
You may need to also run `cargo update -p serial_test_derive:0.6.0` to solve this, since it looks like `mc-consensus-service` is still using the older derive crate.

Reply:
we actually don't need this library anymore, as the `serial_test` is now containing the `serial` macro.